letsencrypt/client/configuration.py

```diff
@@ -48,6 +48,18 @@ def cert_key_backup(self):  # pylint: disable=missing-docstring
             self.namespace.work_dir, constants.CERT_KEY_BACKUP_DIR,
             self.namespace.server.partition(":")[0])
 
+    @property
+    def accounts_dir(self):  #pylint: disable=missing-docstring
+        return os.path.join(
+            self.namespace.config_dir, constants.ACCOUNTS_DIR,
+            self.namespace.server.partition(":")[0])
```

Comment:
`partition` might cause us a headache if there are multiple ACME instances under the same domain, but different ports or paths (c.f. https://github.com/letsencrypt/lets-encrypt-preview/pull/362/files#r29230777)

how about `replace(':', '-').replace('/', '-')` and an appropriate TODO comment?